covalent/_workflow/depscall.py

```diff
@@ -23,6 +23,8 @@
 from .deps import Deps
 from .transport import TransportableObject
 
+RESERVED_RETVAL_KEY__FILES = "files"
```

Comment:
I wonder if a less generic name would be appropriate here, like `file_transfer_files`. Electrons might implement file processing logic in addition to that provided by file transfer.

Reply:
The `files` keyword was chosen to be concise as users will have to use this kwarg everywhere there is a file transfer to get the abstracted filepaths I think making it longer would degrade user experience. Could you maybe elaborate on what type of file processing logic would occur so we can weigh the importance and if this would also be a user facing feature?
cheers @cjao

Reply:
This is mostly a hypothetical concern and we can revisit it down the road if it does become a problem.